src/IO/Archives/LibArchiveReader.cpp

```diff
@@ -185,6 +240,8 @@ class LibArchiveReader::Handle
     }
 
     const String path_to_archive;
+    std::unique_ptr<StreamInfo> read_stream = nullptr;
```

Comment:
It's default constructed to `nullptr`, so you could just go simpler.
```suggestion
    std::unique_ptr<StreamInfo> read_stream;
```